test/integration/operator_olm_test.go

```diff
@@ -267,6 +269,19 @@ func PackageManifestsMultiplePackages(t *testing.T) {
 	assert.NoError(t, doUninstall(t, kubeconfigPath))
 }
 
+// TODO: After rewritting the tests to ginko/gomega format, the test context will be created
+// at the beginning of the suite.
+func TestOLMInstallFromBindata(t *testing.T) {
```

Comment:
Im not sure if a new test context is required though. The current test context uses the image `"OSDK_INTEGRATION_IMAGE"`. Is olm is installed with it?